web/src/routes/UsageLogs/UsageLogsTable.tsx

```diff
@@ -92,7 +92,7 @@ export function UsageLogsTable(props: UsageLogsTableProps) {
           <Table variant="compact" borders={false} style={{margin: '20px'}}>
             <Thead>
               <Tr>
-                <Th>Date & Time</Th>
+                <Th width={15}>Date & Time</Th>
```

Comment:
Just curious, why does a specific width need to be set for this header?

Reply:
The idea was to stop splitting the date and time column across multiple lines. Because this split occurs on random places (for instance, sometimes letters `AM` and `PM` are moved to the 2nd line, while the rest of the date is left on the first line). This doesn't really look good and is occurring consistently, especially when larger descriptions are in place and the browser window is zoomed. I thought that maybe setting the width to a fixed size will stop that but I guess something else needs to be added to achieve it.

Reply:
@Marcusk19 I added a truncate option on the text field, that should keep the `date/time` always on one line and will truncate it if zoom levels are too high so that text cannot be displayed in the defined column width.